greetings-tycho/2.26.0/org.xtext.example.mydsl/META-INF/MANIFEST.MF

```diff
@@ -7,6 +7,9 @@ Bundle-SymbolicName: org.xtext.example.mydsl; singleton:=true
 Bundle-ActivationPolicy: lazy
 Require-Bundle: org.eclipse.xtext,
  org.eclipse.xtext.xbase,
- org.eclipse.equinox.common;bundle-version="3.5.0"
+ org.eclipse.equinox.common;bundle-version="3.5.0",
+ org.eclipse.emf.mwe2.launch;resolution:=optional,
```

Comment:
am not sure if we want this. 
this introduces a dependency that might be resolved depending
on how the plugin is installed.

Reply:
question: we use tycho 1.7 for java 8 and tycho 2.3 for java 11.
can you check if tycho 2.3 can do the additional bundles stuff?

Reply:
yes, I know, that's why probably the other solution for the exec plugin was chosen...
I don't think having that optional dependency harms in any case...
but I'm also testing an alternative solution...

Reply:
yes, the alternative solution works as well and it doesn't touch the MANIFEST.. of course it touches the POM https://github.com/itemis/xtext-reference-projects/pull/183

Reply:
its actually a tycho 2.4 feature 
https://wiki.eclipse.org/Tycho/Release_Notes/2.4